File: csharp-beginner-workbook/module-02-basics/examples/example-02-user-input.cs
// Example 2: Getting User Input
// This demonstrates how to get input from the user and use it

Console.Write("What's your name? ");
string name = Console.ReadLine();

Console.Write("How old are you? ");
string ageText = Console.ReadLine();
int age = int.Parse(ageText);

Console.WriteLine($"Hello, {name}! You are {age} years old.");

File: python-beginner-workbook/module-02-basics/examples/example-02-user-input.py
# example-02-user-input.py
# Demonstrates getting input from users

# Get user's name (always returns a string)
name = input("Enter your name: ")

# Get user's age (convert string to integer)
age = int(input("Enter your age: "))

# Get a price (convert string to float)
price = float(input("Enter a price: £"))

# Display the collected information
print("\n=== Information Summary ===")
print(f"Name: {name}")
print(f"Age: {age}")
print(f"Price: £{price:.2f}")

# Calculate something with the input
years_until_100 = 100 - age
print(f"\nYou'll be 100 years old in {years_until_100} years!")

